bin/run_processor.sh

```diff
@@ -4,18 +4,19 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-# Usage: bin/run_processor.sh
+# Usage: bin/run_processor_honcho.sh
 #
-# Runs the processor.
+# Runs processor process manager.
 #
 # Note: This should be called from inside a container.
 
 set -euo pipefail
 
-CMDPREFIX="${CMDPREFIX:-}"
+PROCESSOR_WORKERS=${PROCESSOR_WORKERS:-"1"}
```

Comment:
We're setting this to 1 for now because the processor is still a multi-threaded application. In a future PR for bug 907277, we'll change it from a multi-threaded app to a multi-process app.